### PR TITLE
Add icon margin that works on both LTR & RTL, re-add removed styles

### PR DESCRIPTION
**Pull request by sheela-svmx**
Issue: 

- The -webkit and -moz fix would repleace margin-left / margin-right depending on the direction of render. But this fix does not work on IE plus unnessary browser prefix in the code.

- Styles were removed in this PR
This was causing regression in px-dropdown padding left when it was used in readOnly mode.

Proposed new:

Add margin-right as well , so icon will have margin-right and margin-left, so irrespective of the rendering, text would not be too close to the icon 

Re-add back the removed styles

screen shot 2019-03-05 at 10 40 32 am

screen shot 2019-03-05 at 10 52 42 am

screen shot 2019-03-05 at 10 41 10 am

### CODE REVIEW

**Comment by sheela-svmx** [comment]
@benjaminliugang @joshsylvester

File: sass/px-dropdown.scss [hunk 1]
```diff
@@ -62,13 +62,7 @@ $inuit-enable-btn--primary: true;
...
   --iron-icon-height: 1.06667rem;
   margin-left: $inuit-base-spacing-unit--tiny;
-  /*
-    These margins are for pages that are rendered from "Right-To-Left" or "Left-To-Right"
-    If the writing direction is left-to-right, this value overrides margin-left.
-    If the writing direction is right-to-left, this value overrides margin-right.
-  */
-  -webkit-margin-start: $inuit-base-spacing-unit--tiny;
-  -moz-margin-start: $inuit-base-spacing-unit--tiny;
+  margin-right: $inuit-base-spacing-unit--tiny;
```
Comment (benjaminliugang) [inline sass/px-dropdown.scss]: This fix only can add margin-right value for the dropdown button icon's, the item icon's space issue still exist.
image

Comment (sheela-svmx) [reply sass/px-dropdown.scss]: @joshsylvester  and @benjaminliugang  updated

File: HISTORY.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# v4.9.3
+
+- Use margin right and left to avoid browser specif prefix fix for rtl space issue
```
Comment (evanjd) [inline HISTORY.md]:
```suggestion
- Use margin right and left to avoid browser specific prefix fix for rtl space issue
```

**Comment by evanjd** [comment]
Also - just repeating again my _strong_ preference for not committing version bumps and HISTORY.md updates in PRs - as both this PR and #105 will create conflicts for each other.

Also, judging from the comments, both this and PR #105 treat the same problem? They should probably be part of a single version bump rather than released individually.

**Comment by sheela-svmx** [comment]
@evanjd  Agreed I will combine the 2 PR into 1 so there will be one version change

**Comment by sheela-svmx** [comment]
@evanjd  updated ur comments and combined into one PR. will close the other PR.

**Pull request merged**